test/e2e/page-objects/pages/confirmations/redesign/token-transfer-confirmation.ts

```diff
@@ -45,30 +39,6 @@ class TokenTransferTransactionConfirmation extends TransactionConfirmation {
     await this.driver.clickElement(this.confirmButton);
   }
 
-  /**
-   * Edits the gas fee by setting custom gas limit and price values
-   *
-   * @param gasLimit - The gas limit value to set
-   * @param gasPrice - The gas price value to set
-   */
-  async editGasFee(gasLimit: string, gasPrice: string): Promise<void> {
```

Comment:
this function has been moved to the parent class `TransactionConfirmation` because it's not exclusive from the Token confirmation, but rather belongs to its parents, to be used with other tests